test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -219,7 +226,8 @@ context('action sidebar', function() {
       .wait('@routePatientActions')
       .wait('@routeAction')
       .wait('@routeActionActivity')
-      .wait('@routePatient');
+      .wait('@routePatient')
+      .wait('@routeGroups');
```

Comment:
I doubt this wait is necessary?  None of the routes above will run until this one occurs because they all wait on the bootstrap to be successful.  It's kind of implicitly `wait`'d